odoo/addons/base/models/ir_qweb.py

```diff
@@ -646,12 +646,12 @@ def _get_template(self, template, options):
             element = template
             document = etree.tostring(template, encoding='unicode')
             ref = None
-
+        # template is falsy
+        elif not template:
+            raise ValueError('Template identifier is falsy')
```

Comment:
I would replace that with a simple `assert template, "template is required"` at the top of the method, as it's not really supposed to happen, right? Or just omit it?